frontend/src/metabase/parameters/components/ParametersList/types.ts

```diff
@@ -0,0 +1,28 @@
+import type {
+  DashboardFullscreenControls,
+  DashboardHideParametersControls,
+  DashboardThemeControls,
+} from "metabase/dashboard/hoc/controls";
+import type Question from "metabase-lib/v1/Question";
+import type { Dashboard, Parameter, ParameterId } from "metabase-types/api";
+
+export type ParametersListProps = {
+  parameters: Parameter[];
+} & Partial<
+  {
+    className?: string;
+    question?: Question;
+    dashboard?: Dashboard | null;
+    editingParameter?: Parameter | null;
```

Comment:
This makes these parameters accept both `undefined` and `null`. If we know that it only accepts `null` and not `undefined` we can also update the type here to be more strict (e.g. `dashboard: Dashboard | null`. Otherwise ok to leave as-is.